src/App.tsx

```diff
@@ -1,19 +1,139 @@
 import React from 'react';
-import './App.css';
+import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/types';
 
 import users from './api/users';
+import todos from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+interface State {
+  todos: Todo[],
+  title: string,
+  select: number,
+  choosenTitle: boolean,
+  choosenPerson: boolean,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    todos: [...todos],
+    title: '',
+    select: 0,
+    choosenTitle: true,
+    choosenPerson: true,
+  };
+
+  addTodo = (event: React.FormEvent) => {
+    event.preventDefault();
+    this.checkErrors();
+    // eslint-disable-next-line @typescript-eslint/no-shadow
+    const { title, select } = this.state;
+
+    if (title && select) {
+      this.setState(state => ({
+        todos: [...state.todos, {
+          title: state.title,
+          id: state.todos.length + 1,
+          userId: state.select,
+        }],
+      }));
+    }
```

Comment:
you could optimize and not let the code go further if there are errors. like an early return statement